salt/version.py

```diff
@@ -289,32 +359,12 @@ def from_name(cls, name):
         return cls(*cls.LNAMES[name.lower()])
 
     @classmethod
-    def from_last_named_version(cls):
```

Comment:
I think the right thing to do here is to put this function on a deprecation path first.

Reply:
Done